conans/client/migrations.py

```diff
@@ -69,34 +69,6 @@ def _make_migrations(self, old_version):
         cache = ClientCache(self.cache_folder, self.out)
         self._update_settings_yml(cache, old_version)
 
-        if old_version < Version("1.0"):
-            _migrate_lock_files(cache, self.out)
```

Comment:
You are leaving all the implementations of these calls as dead code, remove it too.